scripts/commands/checkOrphanPages.ts

```diff
@@ -139,4 +142,67 @@ function collectTocFileContents(children: TocEntry[]): string[] {
   return urls;
 }
 
+function apiDocsIgnores(): string[] {
+  const versions = [
+    "",
+    "dev/",
+    "0.7/",
+    "0.8/",
+    "0.9/",
+    "0.14/",
+    "0.15/",
+    "0.16/",
+    "0.17/",
+    "0.18/",
+    "0.19/",
+    "0.20/",
+    "0.24/",
+    "0.25/",
+    "0.26/",
+    "0.27/",
+    "0.28/",
+    "0.29/",
+    "0.30/",
+    "0.31/",
+    "0.32/",
+    "0.33/",
+    "0.35/",
+    "0.36/",
+    "0.37/",
+    "0.38/",
+    "0.39/",
+    "0.40/",
+    "0.41/",
+    "0.42/",
+    "0.43/",
+    "0.44/",
+    "0.45/",
+    "0.46/",
+  ];
+
+  return flattenDeep([
+    versions.map((vers) => [
```

Comment:
Instead, use `.flatMap()` here. Then you can remove `flattenDeep` :)

Reply:
Good suggestion, thanks!